SPTDataLoader/SPTDataLoader.m

```diff
@@ -71,9 +72,18 @@ - (void)executeDelegateBlock:(dispatch_block_t)block
 {
     SPTDataLoaderRequest *copiedRequest = [request copy];
     id<SPTDataLoaderDelegate> delegate = self.delegate;
+
+    // Cancel the request immediately if it requires chunks and the delegate does not support that
     if ([delegate respondsToSelector:@selector(dataLoaderShouldSupportChunks:)] && copiedRequest.chunks) {
```

Comment:
Shouldn't you now also message the delegate in the if-statement to check if it supports chunks?